examples/scatterplots.js

```diff
@@ -0,0 +1,41 @@
+import createPilingJs from '../src/library';
+import createScatterplotRenderer from './scatterplot-renderer';
```

Comment:
Just a note, I moved the renderer out of `src`. We should only keep general renderers that can be used in multiple cases in the main source code.